contracts/svm/testing/tests/submit_bid.rs

```diff
@@ -49,15 +46,8 @@ pub struct BidInfo {
     pub ixs:                  Vec<Instruction>,
 }
 
-pub const SPLIT_ROUTER_DEFAULT: u64 = 4000;
-pub const SPLIT_RELAYER: u64 = 2000;
-
 fn setup_bid() -> BidInfo {
-    let setup_result = setup(SetupParams {
-        split_router_default: SPLIT_ROUTER_DEFAULT,
-        split_relayer:        SPLIT_RELAYER,
-    })
-    .expect("setup failed");
+    let setup_result = setup(None).expect("setup failed");
```

Comment:
slight nit here i would also specify the split params here since that factors into the test logic

Reply:
I disagree, this test works with any values of split_relayer and split_router_default